prow/config.yaml

```diff
@@ -1769,6 +1833,63 @@ presubmits:
         hostPath:
           path: /mnt/disks/ssd0
 
+  - name: pull-kubernetes-node-e2e-containerd
+    agent: kubernetes
+    branches:
+    - master
+    always_run: false
+    skip_report: true
+    max_concurrency: 8
+    context: pull-kubernetes-node-e2e-containerd
+    rerun_command: "/test pull-kubernetes-node-e2e-containerd"
+    trigger: "(?m)^/test pull-kubernetes-node-e2e-containerd,?(\\s+|$)"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20171108-8b5aad9f-master
+        args:
+        - --root=/go/src
+        - "--clean"
+        - "--git-cache=/root/.cache/git"
+        - "--job=$(JOB_NAME)"
+        - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+        - "--repo=github.com/kubernetes-incubator/cri-containerd"
+        - "--service-account=/etc/service-account/service-account.json"
+        - "--upload=gs://kubernetes-jenkins/pr-logs"
+        - "--timeout=90"
+        - "--"
+        - "--node-args=--image-config-file=test/e2e_node/image-config.yaml --image-config-dir=$GOPATH/src/github.com/kubernetes-incubator/cri-containerd"
```

Comment:
same thing here, since the rest of the job config is in config.json and we don't need workarounds for older releases / jenkins just inline the scenario args into config.json for now please

Reply:
Done.